vertical-pod-autoscaler/pkg/admission-controller/logic/limit_range_calculator.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logic
+
+import (
+	"fmt"
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/informers"
+
+	v1_listers "k8s.io/client-go/listers/core/v1"
+)
+
+// LimitsRangeCalculator checks for LimitRange and if container needs limits to be set
+type LimitsRangeCalculator interface {
```

Comment:
This is actually a LimitRangeItemLister, with a filter applied. So maybe LimitRangeFetcher would be a more appropriate name.

Reply:
Ok, then since it's LimitRange not LimitsRange, lets call it LimitRangeCalculator